vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -3,178 +3,157 @@ sidebarDepth: 2
 ---
 # Run Blueprint-generated Microservices and Micro Frontends in Dev Mode
 ## Overview
-This tutorial will take you through running an Entando project (microservice and micro frontends) in a local development environment. If you haven't generated your Entando project yet, start with the [Generate Microservices and Micro Frontend](./generate-microservices-and-micro-frontends.md) tutorial first.
+This tutorial guides you through running an Entando project with microservice and micro frontends in a local development environment. If you haven't generated an Entando project yet, start with the [Generate Microservices and Micro Frontend](./generate-microservices-and-micro-frontends.md) tutorial.
 
```

Comment:
As we discussed, add a note here explaining this tutorial is specific to bundles based on the JHipster Blueprint with a link to v1 versus v5 language.